ionic/src/app/add-employee/add-employee.component.ts

```diff
@@ -14,14 +14,5 @@ export class AddEmployeeComponent implements OnInit {
 
   ngOnInit() {
   }
-  async setEmployeeID(devices: Device) {
```

Comment:
for now i remove employeID function because at that time i don't have employee id and chip device for particular employee